app/controllers/apicasso/apidocs_controller.rb

```diff
@@ -23,6 +24,7 @@ class ApidocsController < Apicasso::ApplicationController
           key :name, I18n.t('application.license')
         end
       end
+      # Definitions of models from Application
```

Comment:
Improve description of "Definitions of models from Application", what is being defined and for what purpose?